file-formats/susi/type/zif_aff_susi_v1.intf.abap

```diff
@@ -0,0 +1,175 @@
+      "! <p class="shorttext">Application Type</p>
+      "! Type of Authorization Default Value.
+      "! $required
+      type             TYPE ty_type,
+      "! <p class="shorttext">Technical Name (Hash Code)</p>
+      "! Technical Name (Hash Code).
+      "! $required
+      hash_name        TYPE ty_hash_name,
+      "! <p class="shorttext">Maintenance Mode</p>
+      "! Maintenance Mode.
+      "! $values {@link if_aff_susi_v1.data:co_maintenance_mode}
+      "! $required
+      maintenance_mode TYPE ty_maintenance_mode,
+    END OF ty_general.
+
+  TYPES:
+    "! <p class="shorttext">Authorization Value</p>
+    "! Authorization Value
+    ty_val TYPE c LENGTH 40.
+
+  TYPES:
+    "! <p class="shorttext">Authorization Field</p>
+    "! Authorization Field
+    ty_field TYPE c LENGTH 10.
+
+  TYPES:
+    "! <p class="shorttext">Authorization Field Details</p>
+    "! Authorization Field Details
+    BEGIN OF ty_authorization_field,
+      "! <p class="shorttext">Field</p>
+      "! Name of Authorization Field
+      "! $required
+      field TYPE ty_field,
+      "! <p class="shorttext">From</p>
+      "! From Value
+      "! $showAlways
+      low   TYPE ty_val,
+      "! <p class="shorttext">To</p>
+      "! To Values
+      "! $showAlways
+      high  TYPE ty_val,
+    END OF ty_authorization_field.
+
+  TYPES:
+    "! <p class="shorttext">Authorization Fields</p>
+    "! Authorization Fields
+    ty_authorization_fields  TYPE STANDARD TABLE OF ty_authorization_field WITH DEFAULT KEY.
+
+  TYPES:
+    "! <p class="shorttext">Authorization Objects Details</p>
+    "! Authorization Objects Details
+    BEGIN OF ty_authorization_object,
+      "! <p class="shorttext">Object</p>
+      "! Name of Authorization Object
+      "! $required
+      object             TYPE c LENGTH 10,
+      "! <p class="shorttext">Maintenance Status</p>
+      "! Maintenance Status
+      "! $values {@link zif_aff_susi_v1.data:co_maintenance_status}
+      "! $default {@link zif_aff_susi_v1.data:co_maintenance_status.y_flag}
+      "! $showAlways
+      maintenance_status TYPE ty_maintenance_status,
+      "! <p class="shorttext">Fields</p>
+      "! Authorization Fields.
+      "! $showAlways
+      fields             TYPE ty_authorization_fields,
+
+    END OF ty_authorization_object.
+
+  TYPES:
+  "! <p class="shorttext">Authorization Objects</p>
+  "! Authorization Objects
+    ty_authorization_objects TYPE STANDARD TABLE OF ty_authorization_object WITH DEFAULT KEY.
+
+  TYPES:
+    "! <p class="shorttext">SUSI Object Type</p>
+    "! Object type SUSI
+    BEGIN OF  ty_main,
+      "! $required
+      format_version        TYPE zif_aff_types_v1=>ty_format_version,
+      "! <p class="shorttext">Header</p>
+      "! Header
+      "! $required
+      header                TYPE zif_aff_types_v1=>ty_header_60_src,
+      "! <p class="shorttext">General</p>
+      "! General
+      "! $required
+      general               TYPE ty_general,
```

Comment:
```suggestion
      general_information               TYPE ty_general_information,
```

P.S. For consistency reasons I would also rename the type. However, this is not mandatory.

Reply:
done

Reply:
done